### PR TITLE
[BLO-790] User can fill in a multisg creation form

### PR DESCRIPTION
**Pull request by diegodelrieu**
### Issue / feature description

Form for multisig creation (without backend call)

### CODE REVIEW

File: packages/extension/scripts/hot-reload-server.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import path from "path"
 
 import chokidar from "chokidar"
-import { debounce } from "lodash-es"
+import { debounce } from "lodash"
```
Comment (dhruvkelawala) [inline packages/extension/scripts/hot-reload-server.ts:4]: Why?

File: packages/extension/src/ui/features/accounts/multisig/CreateMultisigScreen/index.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+`
+
+const StyledPageWrapper = styled(PageWrapper)`
+  ${({ theme }) => theme.mediaMinWidth.md`
+    > ${Panel}:last-child {
+      background:url('./assets/onboarding-background.svg') no-repeat center;
+      background-size: cover;
+    }
+  `}
+`
```
Comment (dhruvkelawala) [inline packages/extension/src/ui/features/accounts/multisig/CreateMultisigScreen/index.tsx:58]: We are moving away from styled-components and sticking with chakra-ui & argent-ui for everything new

**Review by dhruvkelawala — CHANGES_REQUESTED** [review]

**Comment by dhruvkelawala** [comment]
Screenshot 2023-02-20 at 11 44 15

I think it's better to validate "incorrect signer" on submit (not onBlur), because user might be waiting for other signers and I don't think we should show error message at that time

File: packages/extension/src/ui/features/accounts/multisig/CreateMultisigScreen/index.tsx (hunk 2 above)
Comment (diegodelrieu) [reply packages/extension/src/ui/features/accounts/multisig/CreateMultisigScreen/index.tsx:58]: Ok yeah this was just lazy copy pasta from a similar page but will migrate to chakra

**Review by dhruvkelawala — APPROVED** [review]

**Pull request merged**